VisualStudio/oqs/oqs.vcxproj.filters

```diff
@@ -7,9 +7,6 @@
     <ClCompile Include="..\..\src\common\rand.c">
       <Filter>common</Filter>
     </ClCompile>
-    <ClCompile Include="..\..\src\common\rand_nist.c">
-      <Filter>common</Filter>
-    </ClCompile>

```

Comment:
Why is this removed?

Reply:
This was added in PR #348, and my PR was prepared before that landed. I'll merge with the latest.